package.json

```diff
@@ -24,11 +24,13 @@
   },
   "homepage": "https://github.com/poooi/plugin-quest#readme",
   "dependencies": {
-    "inflection": "^1.12.0"
+    "inflection": "^1.12.0",
+    "kcwiki-quest-data": "github:kcwikizh/kcwiki-quest-data#release"
```

Comment:
This should be marked dev or peer dependencies, otherwise all users who install this plugin will have to install it, which would cause lots of failure because git is not installed

Reply:
from the usage I feel it should be a git submodule, or we need to figure out how to bundle the plugin to prevent its dependencies being installed

Reply:
Release to GitHub just for convenience.
npm package should be simpler than git submodule or bundle, I think.

It seem I need to publish it to npm later.